cmd/guacgql/cmd/ent.go

```diff
@@ -0,0 +1,46 @@
+//
+// Copyright 2023 The GUAC Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build !(386 || arm || mips)
+
+package cmd
+
+import (
+	"context"
+
+	"github.com/guacsec/guac/pkg/assembler/backends"
+	entbackend "github.com/guacsec/guac/pkg/assembler/backends/ent/backend"
+)
+
+func init() {
```

Comment:
ahh i wasn't thinking of it this way, but this works as well - i actually thought of doing this in the ent backend package itself.

Reply:
Yeah, even if the backend is ignored, the cmd can't have any code that pulls it in, so just do it here.